tasks/depcheck.js

```diff
@@ -45,6 +47,20 @@ module.exports = function (grunt) {
           });
         }
 
+        if (unused.missing && Object.keys(unused.missing).length !== 0) {
+          fail = options.failOnMissingDeps;
+          grunt.log.warn('Missing Dependencies');
+          Object.keys(unused.missing).forEach(function (u) {
```

Comment:
Please give a more meaningful name to `u` variable, e.g., `missingDependency`.

---

I feel sad that, this kind of short names are widely used in this file. 😢 

@nboxhallburnett What is your idea about this naming style?